lib/constants.ts

```diff
@@ -1,3 +1,5 @@
+export const entriesPerPage = 25;
```

Comment:
I wonder if we should use upper case snake case for the consts:

```suggestion
export const ENTRIES_PER_PAGE = 25;
```